### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by aljones15**
1. Adds a rule so arrow functions can have a single parameter with no parenthesis
2. Creates standalone vue rule set other projects can use

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-digitalbazaar",
-  "version": "0.0.1-0",
+  "version": "0.0.1",
```
Comment (mattcollier) [inline package.json]: @aljones15 I think you need to rebase on master, some of the changes here like to the readme are already on master.  Also, the publish script takes care of bumping versions, so there is no need to modify the package.json in dev branches.

Comment (aljones15) [reply package.json]: yeah I thought that Pr comment was weird too with changing the version.

Comment (aljones15) [reply package.json]: btw master currently 0.0.1-0 which I believe will cause issues when bumping versions.

Comment (mattcollier) [reply package.json]: @aljones15 maybe you need to pull master?  https://github.com/digitalbazaar/eslint-config-digitalbazaar/blob/master/package.json#L3

**Pull request closed without merging**